internal/state/helpers.go

```diff
@@ -131,6 +135,10 @@ func DisplayProgressCircle(cmd *cobra.Command, errCh <-chan error, waitingFor st
 		ellipsis = " ... "
 	)
 
+	if quiet, _ := cmd.Flags().GetBool("quiet"); quiet {
+		return <-errCh
```

Comment:
```suggestion
		return nil
```

Reply:
Why should we not wait for the action to finish?

Reply:
Woups, you are right.